common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -567,11 +565,6 @@ public boolean isExpired() {
             return state() == ST_EXPIRED;
         }
 
```

Comment:
line 537 there is a comment which references MpscLinkedQueueNode ... plz remove.